src/common/s3.ts

```diff
@@ -128,11 +128,12 @@ export async function deleteObjectsByPrefix(
   prefix: string,
 ) {
   let delCount = 0;
+  let listObjectsCommand = new ListObjectsV2Command({
```

Comment:
yes. It lists all objects in a s3 bucket with a specific prefix, should use prefix to restrict objects.

Reply:
actually, I added this function to do some clean work, but finally I did not use the function in any code(changed to use spark write overwrite mode),but still kept the function there for deleting objects under a prefix, it may be helpful for cleaning some folders in S3 bucket.